samples/nrf9160/serial_lte_modem/README.rst

```diff
@@ -141,6 +141,15 @@ The following proprietary GPS AT commands are used in this sample:
 
 * AT#XGPS=<op>[,<mask>]
 
+MQTT AT Commands
+****************
+
+The following proprietary MQTT AT commands are used in this sample:
+
+* AT#XMQTTCON=<op>[,<url>,<port>[,<sec_tag>]]
+* AT#XMQTTPUB=<topic>,<datatype>,<msg>,<qos>,<retain>
+* AT#XMQTTSUB=<topic>,<op>[,<qos>]
```

Comment:
Descriptions on #XMQTTEVT and #XMQTTMSG are added to the document.

Reply:
> How about AT#XMQTTSUB=[,] and AT#XMQTTUNSUB=, as SUBSCRIBE and UNSUBSCRIBE are standard MQTT control packets and commonly used, so it would be good to have two explicit commands

AT#XMQTTSUB is implemented.
Syntax of subscribe/unsubscribe are:
* AT#XMQTTSUB=<topic>,<qos>
* AT#XMQTTUNSUB=<topic>